docs/readme/sample_pipelines.md

```diff
@@ -0,0 +1,136 @@
+# Data Prepper Tutorial
```

Comment:
This feels a lot like getting started and not a simple_pipelines.md file.

Reply:
I originally called it "Tutorial" and accidentally left that terminology in the header.

I'm happy to call it "Sample Pipeline", "Tutorial", or something else similar.

Reply:
Also, I put this into a separate file because I envision that after we have log analytics, we will have three "Getting Started" pipeline documents:
1. Trace Analytics
2. Log Analytics
3. Simple Pipeline.

I can also put all of this into the main "Getting Started" if you think that would be better.

Reply:
I really think we have two guides:
1. Getting Started
    1. Observability (Trace and Log Analytics)
    1. Simple Pipeline
1. Tutorial
    1. Building pipelines
    1. Scaling data prepper

Then we have another document about scaling and tuning.

Reply:
OK. Based on that comment, I replaced "Tutorial" with "Simple Pipeline."

And now, that does give this structure current:

1. Getting Started
 i. Trace Analytics
 ii. Simple Pipeline